lib/src/command/update_sdk_command.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:moronepo/src/command/moronepo_command.dart';
+import 'package:moronepo/src/flutter_finder/flutter_finder.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:quiver/check.dart';
+
+import '../project_finder/project.dart';
+import '../project_finder/project_finder.dart';
+
+class UpdateSdkCommand extends MoronepoCommand<Null> {
+  @override
+  String get description =>
+      "Updates Flutter SDK version to environment.flutter in root pubspec.yaml";
+
+  @override
+  String get name => "update-sdk";
```

Comment:
What do you think be more explicit and named it "update-flutter-sdk"? Moronepo can be used in non flutter project.